src/main/webapp/js/jsUtils.js

```diff
@@ -98,6 +98,8 @@ var mockShowEndorsementResponse = function(){
         
         response.endorsed.name = names[i-1];
         response.endorsed.photo = "https://storage.googleapis.com/tech-gallery-assets/userPhotos/user" + i + ".jpg";
+        response.endorsed.clientId = i;
+        response.endorsed.email = response.endorsed.name+"@example.com";
```

Comment:
It's inside a mock method, just for example. In the future the full method can be removed.